iOS Client/ArticlesController/MBArticlesViewController.swift

```diff
@@ -15,6 +15,7 @@ class MBArticlesViewController: UIViewController, UITableViewDelegate, UITableVi
     var articlesByCategory = [String: [MBArticle]]()
     var topLevelCategories: [String] = []
     let client = MBClient()
+    let articlesStore = MBArticlesStore()
```

Comment:
This is a good call so we don't have to instantiate new stores everywhere. One thing we do on Synchrony is inject these dependencies via a "build" method which would be our version of instantiateFromStoryboard() since we don't have inits for these view controllers. Might be something to consider for a future refactor